lib/web/css/source/lib/_typography.less

```diff
@@ -63,8 +63,16 @@
     .lib-css(font-size, @fontValue);
 }
 
-//  Rem line height
-.lib-line-height(@heightValue) when not (@heightValue = false) and not (@heightValue = normal) and not (ispercentage(@heightValue)) {
+//  Lib line-height
+//  Support unitless. Ex: .lib-line-height(1.5) => line-height: 1.5
+//
+//  _____________________________________________
+
+.lib-line-height(@heightValue) when not (@heightValue = false) and not (@heightValue = normal) and not (ispercentage(@heightValue)) and (isnumber(@heightValue)) {
+    .lib-css(line-height, @heightValue);
+}
+
+.lib-line-height(@heightValue) when not (@heightValue = false) and not (@heightValue = normal) and not (ispercentage(@heightValue)) and not (isnumber(@heightValue)) {
     .lib-font-size-value(@heightValue);
     .lib-css(line-height, @fontValue);
 }
```

Comment:
My local environments need a tune up before I can test this, but this looks promising! Can you make sure to add some comments explaining what all these booleans are doing? No future dev will be able to make heads or tails of these, speaking from experience looking at M2 LESS 😆